### PR TITLE
Stories: Upgrade Kanvas version

### LINKED ISSUE #18550: WPiOS Crashed When Adding Media to a Story Post
A user reported that when they add media to a Story post, it "blacks out and kicks me out of the app". 

I found the following error in the app logs:

`2022-05-07 19:26:50:197 🔵 Tracked: media_service_upload_failed <default_tab_experiment: nonexistent, error_code: 3, error_description: Error Domain=WordPressKit.WordPressComRestApiError Code=3 "Unable to create directory wp-content. Is its parent directory writable by the server?"
`

I also found a [Sentry report](https://sentry.io/organizations/a8c/discover/wordpress-ios:6f5021e7816f425ea55c98f3f46c0f7b/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&project=1438083&query=user.email%3Aggdollprincess%40gmail.com&sort=-timestamp&statsPeriod=7d&yAxis=count%28%29) with the following error:

```
Deleting a media object that's failed to upload because of a missing local file.
```

##### User/device details
Ticket link: 5198098-zen
App version: 19.7
Device model: iPhone 13 mini (iPhone14,4)
OS version: iOS 15.4.1
Type of site (Simple, Atomic, Jetpack, or self-hosted): Simple

### PR DESCRIPTION
**Pull request by twstokes**
## Description

This PR upgrades Kanvas to version `1.4.3` which fixes a couple issues:
- Memory consumption when exporting multiple images [[issue 1](https://github.com/tumblr/kanvas-ios/pull/140), [issue 2](https://github.com/tumblr/kanvas-ios/pull/141)]
- Retains text layers if publishing is cancelled [[related issue](https://github.com/tumblr/kanvas-ios/pull/141)]
- Fixes #18550

### App doesn't crash with many images
1. Using WordPress / Jetpack iOS, tap the FAB button to create a new Story post
2. Add 15+ images
3. Tap the arrow at the top right to start the publishing process
4. Expect no crash
5. Enter a title and tap "Publish Now"
6. Expect that publishing succeeds and the post can be loaded in the app and on the web (note: the amount of time before the post is published will depend on how many media items were chosen, their size, and the device's network connection speed)

### Text layers aren't lost if publishing is cancelled
1. Using WordPress / Jetpack iOS, tap the FAB button to create a new Story post
2. Take a picture with the camera or add images from the media library
3. Add text to the current item shown (tap the Aa button at the bottom left)
4. Tap the arrow at the top right to start the publishing process
5. Cancel publishing by swiping down on the sheet
6. Expect that the text added remains on the view

## Known issue

No output is shown after cancelling the publishing process and trying to add a photo from the camera.

1. Add media
2. Tap the arrow at the top right to start the publishing process
3. Cancel publishing by swiping down on the sheet
4. Tap the "+" at the bottom right to capture media from the camera
5. Observe a gray screen and no camera output

A Github issue will be added if it doesn't already exist.

## Regression Notes
1. Potential unintended areas of impact
    - Publishing Stories
    - Adding media to a story
    - Adding text layers to media

2. What I did to test those areas of impact (or what existing automated tests I relied on)
    - Manual tests

3. What automated tests I added (or what prevented me from doing so)
    - None

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/Site Picker/SiteIconPickerView.swift
Comment (twstokes) [inline WordPress/Classes/ViewRelated/Blog/Site Picker/SiteIconPickerView.swift:10]: Just FYI - the Site Icon Picker is not currently in production, so these low-risk changes won't affect anything.

**Comment by twstokes** [comment]
👋 @SiobhyB! I'm awaiting reviews on the Kanvas side, but I thought I'd add you as a reviewer in case you wanted to look at things ahead of time. Thanks!

**Review by SiobhyB — APPROVED** [review]
Thank you for this @twstokes 🙇‍♀️ 

I went through the testing steps you outlined, the [story block test cases](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/story.md), and tried to break things through random steps. All looks good to me, with the original bugs squashed!

I know you're still waiting on an approval for https://github.com/tumblr/kanvas-ios/pull/140, but wanted to share my testing so far and clear the way for you for when that PR gets approved. If changes are needed, feel welcome to re-request a review :D

**Pull request merged**
